index.html

```diff
@@ -192,9 +192,9 @@ <h2 class="contact">Contact me</h2>
         to help with it
       </p>
     </div>
-    <form id="contact-form" class="contact-box" action="https://formspree.io/f/mzbolelw" method="POST">
-      <input type="text" name="full-name" placeholder="Your Name" id="contact-name" maxlength="30" required />
-      <input type="email" placeholder="username@email.com" name="email" id="email" required="required" />
+    <form id="contact-form-validation" class="contact-box" action="https://formspree.io/f/mzbolelw" method="POST">
+      <input type="text" name="full-name" placeholder="Your Name" id="contact-name" maxlength="30" required>
+      <input type="email" placeholder="username@email.com" name="email" id="email" required>
       <textarea id="message" name="Message" maxlength="500" placeholder="Write your message here" required></textarea>
       <button type="submit" class="btn btn-width">Get In Touch</button>
     </form>
```

Comment:
Great job @PRATAP-KUMAR on implementing the changes.

- [ ] I could not figure out a way to check your validation as I am getting this message for all submissions. Either with lower case or uppercase.
![Inactive](https://user-images.githubusercontent.com/63917725/180623792-77f52059-6373-4feb-8e1e-a3d1095e246a.PNG)